extras/container/heketi-start.sh

```diff
@@ -6,8 +6,14 @@
 
 : "${HEKETI_PATH:=/var/lib/heketi}"
 : "${BACKUPDB_PATH:=/backupdb}"
+: "${HEKETI_DB_ARCHIVE_PATH:=${HEKETI_PATH}/archive}"
 LOG="${HEKETI_PATH}/container.log"
 
+# allow disabling the archive path
+case "${HEKETI_DB_ARCHIVE_PATH}" in
+    -|/dev/null) HEKETI_DB_ARCHIVE_PATH="" ;;
```

Comment:
Good one!

Reply:
So `:=` applies also to the empty string, not only to undefined variables?